cmd/traffic_layout/info.cc

```diff
@@ -29,22 +29,23 @@
 // Produce output about compile time features, useful for checking how things were built, as well
 // as for our TSQA test harness.
 static void
-print_feature(const char *name, int value, bool json, bool last = false)
+print_feature(ts::string_view const &name, int value, bool json, bool last = false)
```

Comment:
Call me dense, but I thought one of the arguments for using string_view was that it's a simple wrapper around a pointer. As such, is there really value in calling this by reference rather than value?

Reply:
At first, I also thought of this. @dragon512 reminds me it's for safety purpose.

Reply:
Sounds good to me @SolidWallOfCode